Fixed problems with adding new entities to the ldap

NKPmedia opened the pull request:
So that is my fix for the problem.

I did some little tests and it worked for me.
But I think in general it would be good to have some unit tests for this bundle.

I think CarnegieLearningWeb/ldap-orm-bundle did some great work. Maybe it would be nice to merge these to versions or cherry pic some nice commits.

Commit:
Fixed problems with adding new entities to the ldap

NKPmedia commented:
No feedback :| ?

I would be really happy to use your code. :)

jasongabler commented:
NKPmedia,

Slow your roll, man.  I co-opted this from GorgLdapOrm, which was sorely lacking in unit tests. It's really a matter of finding the time to go back and do it for someone else's code.  More importantly, take a look at UcsfRestOrm. It's a ORM I wrote for REST-transfered objects, built upon Doctrine. That is what I want to do for UcsfLdapOrm. If I spend a lot of time on this bundle in the near future, it will be a fourth version similar to that.  As for this bundle's current state, I test the crap out of it in my private bundles which _use_ this bundle. I know that's not orthodox unit testing, but it's what I've got time for right now.  As for merging with Carnegie, what sort of additions have they made that interest you?

NKPmedia committed:
Update services.yml

Not quoting the scalar "%ucsfldaporm_test% " starting with the "%" indicator character is deprecated since Symfony 3.1 and will throw a ParseException in 4.0.

NKPmedia committed:
Update composer.json
